src/NewsService.Business/CreateNewsCommand.cs

```diff
@@ -110,9 +64,7 @@ private List<Guid> CheckDepartmentExistence(Guid? departmentId, List<string> err
 
       OperationResultResponse<Guid?> response = new();
 
-      List<Guid> existDepartments = CheckDepartmentExistence(request.DepartmentId, response.Errors);
-
-      response.Body = _repository.Create(_mapper.Map(request, existDepartments));
+      response.Body = _repository.Create(_mapper.Map(request));
 
       _httpContextAccessor.HttpContext.Response.StatusCode = (int)HttpStatusCode.Created;
 
```

Comment:
you never get errors to the 71 line, you don't need to set a condition in the 71 line

Reply:
done